scripts/local/assets-pull.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+lando pull --code=none --database=test --files=test
```

Comment:
Why does this pull from test?

Reply:
It shouldn't. I didn't look closely enough. I've updated it in the follow-up PR to pull from dev.